tests/angularModuleSpec.js

```diff
@@ -210,12 +210,9 @@ describe('TaskCtl', function () {
     });
 
     it('tests stateMismatch broadcast when state does not match', function () {
-        window.parent.AuthenticationContext = function () {
-            return {
-                callback: function () { },
-                _renewStates: {}
-            };
-        };
+        window.parent.AuthenticationContext = window.AuthenticationContext;
```

Comment:
This creates dependency between 2 tests. So either we should have this line in all the tests below or move it to beforeEach. I am fine with both.